doc/quickstart/configure.rst

```diff
@@ -42,90 +42,95 @@ If the option ``--path`` is omitted, the file will be created in
 ``${HOME}/.esmvaltool``
 
 The following shows the default settings from the ``config-user.yml`` file
-with explanations in a commented line above each option:
+with explanations in a commented line above each option. If only certain values
+are allowed for an option, these are listed after ``---``. The option in square
+brackets is the default value, i.e., the one that is used if this option is
+omitted in the file.
 
 .. code-block:: yaml
 
   # Destination directory where all output will be written
-  # including log files and performance stats
+  # Includes log files and performance stats.
   output_dir: ./esmvaltool_output
 
   # Directory for storing downloaded climate data
   download_dir: ~/climate_data
 
+  # Disable automatic downloads --- [true]/false
   # Disable the automatic download of missing CMIP3, CMIP5, CMIP6, CORDEX,
-  # and obs4MIPs data from ESGF [true]/false.
+  # and obs4MIPs data from ESGF by default. This is useful if you are working
+  # on a computer without an internet connection.
   offline: true
 
-  # Auxiliary data directory, used by some recipes to look for additional datasets
+  # Auxiliary data directory
+  # Used by some recipes to look for additional datasets.
   auxiliary_data_dir: ~/auxiliary_data
 
-  # Rootpaths to the data from different projects (lists are also possible)
-  # these are generic entries to better allow you to enter your own
-  # For site-specific entries, see the default config-user.yml file
-  # that can be installed with the command `esmvaltool config get_config_user`.
-  # For each project, this can be either a single path or a list of paths.
-  # Comment out these when using a site-specific path
+  # Rootpaths to the data from different projects
+  # This default setting will work if automatic downloads are enabled via
```

Comment:
```suggestion
  # This default setting will work if files have been downloaded by the esmvaltool via
```
This setting will work after you have downloaded the files required for running a recipe by using the `--offline=False` command line argument just once. For subsequent runs, the argument is not needed when you set this like this.

To clarify this further: when running the tool with `--offline=False`, the rootpath and drs settings will not really be used, only to check if existing data is present, but there is another check if the files that will be downloaded are already present locally. This is to make sure that even if you do not set this path the same as the download path, the previously downloaded files can still be found.